contracts/Core/StakeManager.sol

```diff
@@ -346,9 +349,11 @@ contract StakeManager is Initializable, ACL, StakeStorage {
         );
         require(stakers[stakerId].id != 0, "staker.id = 0");
 
+
```

Comment:
Hey @adi44 
Its third time we are talking about this
Please remove extra new lines throughout PR and then only request review

Reply:
Hey! Sorry I missed this. I will remove it.